src/test/addrman_tests.cpp

```diff
@@ -52,6 +52,15 @@ class CAddrManTest : public CAddrMan
     {
         CAddrMan::Delete(nId);
     }
+
+    // Simulates connection failure so that we can test eviction of offline nodes
+    void SimConnFail(CService& addr)
+    {
+         int64_t time = 1;
+         Good_(addr, true, time); // Set last good connection in the deep past.
+         bool count_failure = false;
+         Attempt(addr, count_failure);
```

Comment:
Changing this to: `Attempt(addr, count_failure, GetAdjustedTime() - 90);` fixes the unit test failure.

nit: the indentation is off on the line right below